source/reference/config/graphql.txt

```diff
@@ -42,12 +42,13 @@ Service Configuration
 
    * - | ``use_natural_pluralization``
        | Boolean
-     - If ``true``, Atlas App Services uses the common English pluralization of a
-       type name whenever possible.
+     - ``true`` by default on new apps. If ``true``, Atlas App Services uses the common 
```

Comment:
How about this? I think we need to explicitly say here it can't be changed. I'm not sure if I'm changing the meaning:

Defaults to``true`` when creating new apps. You can only set the value to ``false`` when you :admin-api-endpoint:`create a new app with the Admin API endpoint <operation/adminCreateApplication>` and pass the property into the body of the request.  The value cannot be changed to ``false`` once the app is created. 

If ``true``, Atlas App Services uses the common English pluralization of a type name whenever possible. If ``false``, or if App Services cannot determine a natural pluralization, the default plural type is the singular type with an ``"s"`` appended to the end.